src/toolbars/SelectionBar.cpp

```diff
@@ -207,6 +209,14 @@ void SelectionBar::AddTitle(
    pTitle->SetBackgroundColour( theTheme.Colour( clrMedium ));
    pTitle->SetForegroundColour( theTheme.Colour( clrTrackPanelText ) );
    pSizer->Add( pTitle, 0, wxEXPAND | wxRIGHT, 5 );
+
+   pTitle->Bind(
+      wxEVT_LEFT_DCLICK,
+      [this](auto& evt) { SelectionToClipboard(); });
+
+   pTitle->Bind(
+      wxEVT_RIGHT_UP,
+      [this](auto& evt) { SelectionToClipboard(); });
```

Comment:
This is more efficient, as you only need to move the mouse to click once, and the right-click will not lose the focus of the audio selection. Why not let us keep the right click, delete the double click, and then implement a regular operation of Ctrl+C to mTimeControls?

Reply:
If we only were concerned about making timecode copying more efficient, you'd be absolutely right. You even could dedicate a copy button next to the selection settings button to give that functionality some affordance, so it's possible to figure out you even can copy without consulting the manual. 

Unfortunately, we can't really do that. Audacity has millions of users all with different needs and most of them casual. Copying timecodes is not important enough to warrant a permanently visible button, and as we're working on simplifying the app, adding invisible/"you have to read the manual to find it" features is going in the wrong direction. 

The compromise solution here is to allow users which need this function to set themselves one-button shortcuts, and that is something you can do for any menu item:

![image](https://github.com/audacity/audacity/assets/87814144/cdb5f0f7-3d48-47d6-bbf7-7582975a0ca9)

Reply:
Configuring shortcut keys is indeed convenient for keyboard-focused users, but for users who use the keyboard and mouse more balancedly, positioning the mouse on a text label and right-clicking to complete a shortcut function is as convenient as keyboard shortcuts. If possible, I hope to retain the mouse shortcut operation of right-clicking the "Select" text label on the toolbar to activate this function. That position will not cause any side effects, and since there will be a system floating window notify after the operation, it will not cause any confusion to the user.

Reply:
> I'd expect Ctrl+c to copy if one of the NumericTextCtrl boxes is focused.

I'm not sure the user would expect this. They may well be using these controls to fine tune a time selection, and then press Ctrl + C to copy the audio the audio to the clipboard. So I don't think changing the effect of the shortcut here is a good idea.

Reply:
Only when mTimeControls got focus, then they can handle Ctrl+C, and that time, user can not copy audio because focus not on track window.
I just think that the operation of first clicking to select the time control and then pressing Ctrl+C is a bit cumbersome, this requires a combination of mouse and keyboard and cannot be completed with one hand. And it will make the track selection area lose focus.

Reply:
> and that time, user can not copy audio because focus not on track window.

Oh yes they can!

Reply:
> Oh yes they can!

In that case, I also agree to remove Ctrl+C. This is just a small function. It seems to be enough to provide a configurable keyboard shortcut and a mouse shortcut operation entrance.